tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleCore.java

```diff
@@ -255,6 +255,18 @@ public boolean belongsTo( Object family )
 
     private static boolean isWorkspaceWars( IProject project )
     {
+        File possibleWarFile = project.getLocation().toFile().getParentFile();
```

Comment:
Hey @AndyWu2015 this works ok, i'll go ahead and merge this one, but can you send a new pull and improve it by doing the following?

* don't just look in immediate parent.  The `wars` folder can be nested, so you can have a path like this `<workspace>/wars/subdir/sample-jsp-portlet` and it will work ok, so this code needs to first check `is this in a workspace` and then check `is this a subdir of the configured wars folder`
* also the `wars` directory is configurable, the user could do something like this in gradle.properties
```
liferay.workspace.wars.dir=webprojects
```
And now with this type of configuration we need to check for the directory called `webprojects`
* actually the `liferay.workspace.wars.dir` can be a comma separated list, like `liferay.workspace.wars.dir=foo,bar,foobar` and we need to search in all subdirectorys `<workspace>/foo,<workspace>/bar,<workspace>/foobar`
so we would need to check all of those as well.

Thanks